### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Ellipse0934**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,93 @@
...
+- Get the Minecraft: Java edition(MacOS, Linux and Windows) [here](https://minecraft.net/en-us/download/alternative).
+
+### Get RaspberryJuice or RaspberryJamMod (for Minecraft: Java edition)
+
+Officially the ability to communicate with the Minecraft world is only available for the Minecraft: Pi edition. To get this to work on the Java edition we need to install a Mod.
+
+- "RaspberryJamMod" is a Forge Mod, if you wish to use this along with other Forge mods then this is recommended. Installation instructions are available [here](http://www.instructables.com/id/Python-coding-for-Minecraft/).
+- "RaspberryJuice" is a Bukkit server plugin, recommended if you wish to work on a Bukkit Minecraft server. Get it [here](https://dev.bukkit.org/projects/raspberryjuice).
+
+There is a disagreement in the coordinates reported by Minecraft's debug mode and the `player.getPos()` command. Enter `/setworldspawn 0 0 0` in the Minecraft console to remove this discrapency.
```
Comment (aviks) [inline README.md]: Is this true of both Jam and Juice? Would be good to have a sentence here about the underlying cause... `Minecraft Pi sets the spawn point as (0,0,0) but the java version......` etc...

File: README.md [hunk 2]
```diff
@@ -1,33 +1,93 @@
 # PiCraft
 
-### Control a Minecraft world on a Raspberry Pi from Julia
+### Control a Minecraft world on the Minecraft: Pi or Java edition from Julia
 
 ## Installation and Usage
+
+### Get Minecraft
+
+- Get the Minecraft: Pi edition [here](https://minecraft.net/en-us/edition/pi/). It's free!
```
Comment (aviks) [inline README.md]: Maybe mention that is installed in the default image for the Rasbian linux distribution for the Pi

File: README.md (hunk 1 above)
Comment (Ellipse0934) [reply README.md]: True for both. Although there is a setting available in Juice to change b/w absolute and relative coordinate systems. I purposely cut it out. I don't want to increase this document's length at this point. Writing about camera angles, directions and some tips and tricks are at a higher priority but I want to delay this work until I get a better understanding of this project.

File: README.md (hunk 2 above)
Comment (Ellipse0934) [reply README.md]: It's not on Raspbian Jessie and Pi version has been *officially* discontinued. Source: https://www.reddit.com/r/MCPE/comments/42g9cr/is_minecraft_pi_edition_no_longer_being_developed/cza7seq/?context=3&st=jfy7cy9x&sh=12b0b092

Comment (Ellipse0934) [reply README.md]: Although they did say it will come with every Raspbian distro I wouldn't personally count on it.

Comment (Ellipse0934) [reply README.md]: I actually emulated Raspbian Jessie using VirtualBox and this mc version needs an ARM processor, need to do some work to get it to run with an x86 machine. Maybe that's why it didn't show up.

Comment (aviks) [reply README.md]: Yes, while the new minecraft platform edition code does not support the Pi, the existing codebase will continue to be available. Its an important part of the Rasberry Pi ecosystem.

**Pull request merged**